src/App.css

```diff
@@ -1,5 +1,11 @@
 html {
   margin: 0px;
+  overflow: hidden;
+}
+
+main {
+  background-image: url('./assets/background-beach.jpg');
```

Comment:
I like the background image. I think that it might be helpful to have a background color on some of the elements with smaller text so that they are a little easier to read but it doesn't seem high priority.